frontend/src/store.js

```diff
@@ -143,12 +143,12 @@ function useState(path) {
 }
 
 function isAdmin() {
-  let groups = getState(['identity', 'cognito:groups']) || [];
```

Comment:
suggestion(non-blocking): it might be good extract this magic string into at least a constant so that:
- no magic strings are repeated inside the app
- if we choose to adopt properly OAuth2 on the frontend this key will be configured frontend side and thus having that as a constant is better and more easily replaceable by an ENV variable

Reply:
Ok, going to add a `constants.ts` file and set the magic string there